README.md

```diff
@@ -220,3 +220,147 @@ With home_widget you can use this by following these steps:
+    ```swift
+   func getSnapshot(
+        ...
+        let filename = userDefaults?.string(forKey: "filename") ?? "No screenshot available"
+    ```
+3. Create a `View` to display the chart and resize the image based on the `displaySize` of the widget:
+    ```swift
+    struct WidgetEntryView : View {
+      …
+       var ChartImage: some View {
+            if let uiImage = UIImage(contentsOfFile: entry.filename) {
+                let image = Image(uiImage: uiImage)
+                    .resizable()
+                    .frame(width: entry.displaySize.height*0.5, height: entry.displaySize.height*0.5, alignment: .center)
+                return AnyView(image)
+            }
+            print("The image file could not be loaded")
+            return AnyView(EmptyView())
+        }
+    …
+    }
+    ```
+    
+4. Display the chart in the body of the widget's `View`:
+    ```swift
+    VStack {
+            Text(entry.title)
+            Text(entry.description)
+            ChartImage
+        }
+    ```
+
+<img width="522" alt="Screenshot 2023-06-07 at 12 57 28 PM" src="https://github.com/ABausG/home_widget/assets/21065911/f7dcdea0-605a-4662-a03a-158831a4e946">
+
+#### Android
+
+1. Add an image UI element to your xml file:
+    ```xml
+    <ImageView
+           android:id="@+id/widget_image"
+           android:layout_width="200dp"
+           android:layout_height="200dp"
+           android:layout_below="@+id/headline_description"
+           android:layout_alignBottom="@+id/headline_title"
+           android:layout_alignParentStart="true"
+           android:layout_alignParentLeft="true"
+           android:layout_marginStart="8dp"
+           android:layout_marginLeft="8dp"
+           android:layout_marginTop="6dp"
+           android:layout_marginBottom="-134dp"
+           android:layout_weight="1"
+           android:adjustViewBounds="true"
+           android:background="@android:color/white"
+           android:scaleType="fitCenter"
+           android:src="@android:drawable/star_big_on"
+           android:visibility="visible"
+           tools:visibility="visible" />
+    ```
+2. Update your Kotlin code to get the chart image and put it into the widget, if it exists. 
+    ```kotlin
+    class NewsWidget : AppWidgetProvider() {
+       override fun onUpdate(
+           context: Context,
+           appWidgetManager: AppWidgetManager,
+           appWidgetIds: IntArray,
+       ) {
+           for (appWidgetId in appWidgetIds) {
+               // Get reference to SharedPreferences
+               val widgetData = HomeWidgetPlugin.getData(context)
+               val views = RemoteViews(context.packageName, R.layout.news_widget).apply {
+                   // Get chart image and put it in the widget, if it exists
+                   val imageName = widgetData.getString("filename", null)
+                   val imageFile = File("${context.filesDir.path}/${imageName}")
```

Comment:
The `${context.filesDir.path}/` bit should not be necessary as the absolute file path is stored in the widget data.

Reply:
removed it!